TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmode/MainOpMode.java

```diff
@@ -55,12 +53,14 @@ public final class MainOpMode extends BaseOpMode {
     private OpenCvCamera camera;
     private JunctionDetection pipeline;
 
+    protected MainOpMode(boolean usePhoton, boolean useBulkRead) {
```

Comment:
the child classes which implement this constructor should not have any parameters. only super(true/false, true/false)